shard-operator-parent/shard-operator-v2/src/main/resources/templates/camel-integration.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: camel.apache.org/v1
+kind: Integration
+metadata:
+  name: ""
+  labels:
+    app.kubernetes.io/managed-by: bridge-fleet-shard-operator # used for filtering of Deployments created by the controller
```

Comment:
@lucamolteni This should be `smartevents-fleet-shard-operator` to match the `v2` operator name.